[2018-11-01T10:23:42.785] [ERROR] dateFileLog - Error: Not found this user by userAID HMIUser00.
    at callback_getUser (D:\work\Source\yujiang.Foil.Node.BizServer\biz\account\checkLogin.js:106:24)
    at D:\work\Source\yujiang.Foil.Node\src\yjError.js:129:4
    at Object.exports.safeSuccess (D:\work\Source\yujiang.Foil.Node\src\yjError.js:94:13)
    at Object.exports.safeCallback (D:\work\Source\yujiang.Foil.Node\src\yjError.js:128:11)
    at Object.exports.handleResult (D:\work\Source\yujiang.Foil.Node\src\yjError.js:142:11)
    at Query.<anonymous> (D:\work\Source\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:564:33)
    at bound (domain.js:301:14)
    at Query.runBound [as _callback] (domain.js:314:12)
    at Query.Sequence.end (D:\work\Source\node_modules\mysql\lib\protocol\sequences\Sequence.js:96:24)
    at Query._handleFinalResultPacket (D:\work\Source\node_modules\mysql\lib\protocol\sequences\Query.js:143:8)
[2018-11-01T10:23:42.787] [ERROR] dateFileLog - Error: Not found this user by userAID HMIUser00.
    at D:\work\Source\yujiang.Foil.Node\src\yjREST.engine.remote.superagent.js:43:34
    at Request.callback (D:\work\Source\node_modules\superagent\lib\node\index.js:797:3)
    at IncomingMessage.<anonymous> (D:\work\Source\node_modules\superagent\lib\node\index.js:990:12)
    at emitNone (events.js:111:20)
    at IncomingMessage.emit (events.js:208:7)
    at endReadableNT (_stream_readable.js:1064:12)
    at _combinedTickCallback (internal/process/next_tick.js:139:11)
    at process._tickDomainCallback (internal/process/next_tick.js:219:9)
[2018-11-01T10:23:48.687] [ERROR] dateFileLog - Error: Not found this user by userAID HMIUser00.
    at callback_getUser (D:\work\Source\yujiang.Foil.Node.BizServer\biz\account\checkLogin.js:106:24)
    at D:\work\Source\yujiang.Foil.Node\src\yjError.js:129:4
    at Object.exports.safeSuccess (D:\work\Source\yujiang.Foil.Node\src\yjError.js:94:13)
    at Object.exports.safeCallback (D:\work\Source\yujiang.Foil.Node\src\yjError.js:128:11)
    at Object.exports.handleResult (D:\work\Source\yujiang.Foil.Node\src\yjError.js:142:11)
    at Query.<anonymous> (D:\work\Source\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:564:33)
    at bound (domain.js:301:14)
    at Query.runBound [as _callback] (domain.js:314:12)
    at Query.Sequence.end (D:\work\Source\node_modules\mysql\lib\protocol\sequences\Sequence.js:96:24)
    at Query._handleFinalResultPacket (D:\work\Source\node_modules\mysql\lib\protocol\sequences\Query.js:143:8)
[2018-11-01T10:23:48.689] [ERROR] dateFileLog - Error: Not found this user by userAID HMIUser00.
    at D:\work\Source\yujiang.Foil.Node\src\yjREST.engine.remote.superagent.js:43:34
    at Request.callback (D:\work\Source\node_modules\superagent\lib\node\index.js:797:3)
    at IncomingMessage.<anonymous> (D:\work\Source\node_modules\superagent\lib\node\index.js:990:12)
    at emitNone (events.js:111:20)
    at IncomingMessage.emit (events.js:208:7)
    at endReadableNT (_stream_readable.js:1064:12)
    at _combinedTickCallback (internal/process/next_tick.js:139:11)
    at process._tickDomainCallback (internal/process/next_tick.js:219:9)
[2018-11-01T10:26:47.419] [ERROR] dateFileLog - { TypeError: D:\work\Source\toolsWeb\web\app\index.{v}.ejs:10
    8| </style>
    9| <div style="margin:8px">
 >> 10| <% function dumpNodes(nodes){%>
    11| <% for(var i=0;i<nodes.length;i++){ %> 
    12| 	
    13| 	<% if (nodes[i].children && nodes[i].children.length>0){ %>	

Cannot read property 'length' of null
    at dumpNodes (eval at exports.compile (D:\work\Source\node_modules\ejs-locals\node_modules\ejs\lib\ejs.js:237:14), <anonymous>:30:322)
    at eval (eval at exports.compile (D:\work\Source\node_modules\ejs-locals\node_modules\ejs\lib\ejs.js:237:14), <anonymous>:30:1450)
    at eval (eval at exports.compile (D:\work\Source\node_modules\ejs-locals\node_modules\ejs\lib\ejs.js:237:14), <anonymous>:30:1500)
    at D:\work\Source\node_modules\ejs-locals\node_modules\ejs\lib\ejs.js:250:15
    at Object.exports.render (D:\work\Source\node_modules\ejs-locals\node_modules\ejs\lib\ejs.js:288:13)
    at Object.exports.renderFile (D:\work\Source\node_modules\ejs-locals\node_modules\ejs\lib\ejs.js:318:20)
    at View.module.exports [as engine] (D:\work\Source\node_modules\ejs-locals\index.js:85:7)
    at View.render (D:\work\Source\node_modules\express\lib\view.js:126:8)
    at tryRender (D:\work\Source\node_modules\express\lib\application.js:639:10)
    at Function.render (D:\work\Source\node_modules\express\lib\application.js:591:3) path: 'D:\\work\\Source\\toolsWeb\\web\\app\\index.{v}.ejs' }
